deckhouse-controller/entrypoint.sh

```diff
@@ -34,4 +34,37 @@ EOF
 
 cat ${MODULES_DIR}/values-${bundles_map[$bundle]}.yaml >> ${MODULES_DIR}/values.yaml
 
-exec /sbin/tini -- /usr/bin/deckhouse-controller start
+set +o pipefail
+set +e
+
+PID=0
+EXITCODE=0
+
+for SIG in SIGUSR1 SIGUSR2 SIGINT SIGTERM SIGHUP SIGQUIT; do
+  trap "signal_handler ${SIG}" "${SIG}"
+done
+
+signal_handler() {
+  echo "{\"level\":\"warning\", \"msg\": \"Catch signal ${1}\"}"
+  case "${1}" in
+  "SIGUSR1" | "SIGUSR2")
+    kill "${PID}"
+    wait "${PID}"
+    run_deckhouse
+    ;;
+  *)
+    kill -"${1}" "${PID}"
+    ;;
+  esac
+}
+
+run_deckhouse() {
+  /sbin/tini -s -- /usr/bin/deckhouse-controller start &
+  PID="${!}"
+  echo "{\"level\":\"info\", \"msg\": \"New deckhouse PID ${PID}\"}"
+  wait "${PID}"
+  EXITCODE="${?}"
+}
+
+run_deckhouse
+exit "${EXITCODE}"
```

Comment:
This can be omitted, because run_deckhouse is already latest command.
I don't see the problem of seeing this, just see no reason